pages/index.tsx

```diff
@@ -16,7 +19,7 @@ export default function Home() {
           priority
         />
       </div>
-
+      <h1 className="text-3xl p-4">Welcome to Code Youngstown</h1>
```

Comment:
We shouldn't have more than one `h1` tag per page.

I think we should just remove this one entirely.